### ISSUE TITLE
Number of steps per epoch don't match number of batchs in train loader

### ISSUE THREAD

**Issue by nicolas-dufour** [PyTorchLightning/pytorch-lightning#6735]
## 🐛 Bug

When using trainer, the number of steps don't match the number of batch in the train loader.
For example, when using the Boring model, we have 313 batches in the train dataloader, however, when training we get 626 train steps according to the progress bar.

Furthermore, when using the val_check_interval parameter in the trainer we get totally different results.

For val_check_interval = 10 we get 10016 steps, val_check_interval =50 we get 2191 steps and val_check_interval =100 we get 1252 steps

### To Reproduce

Use following [**BoringModel**](https://colab.research.google.com/drive/1SAoKl_I0OtYGxtNYE7OQjNLq3K7VzXrV?usp=sharing) 

### Expected behavior

We expect number of steps to match dataloader length
### Environment

* CUDA:
	- GPU:
		- Tesla T4
	- available:         True
	- version:           10.1
* Packages:
	- numpy:             1.19.5
	- pyTorch_debug:     False
	- pyTorch_version:   1.8.0+cu101
	- pytorch-lightning: 1.2.5
	- tqdm:              4.41.1
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- 
	- processor:         x86_64
	- python:            3.7.10
	- version:           #1 SMP Thu Jul 23 08:00:38 PDT 2020

**Comment by dalek-who** [comment]
The step number in tqdm bar is len(train_dataloader)+epoch_evaluation_num*len(val_dataloader), for example, if len(train_dataloader)=100, len(valid_dataloader)=30, evaluate 3 times each epoch, then step number in tqdm is 100+3*30=190, I think this is a pl feature, but I would like an option to disable this sum up.

**Comment by nicolas-dufour** [comment]
Oh i see thanks, it is quite confusing to have it that way. Indeed, we already have the validation bar to visualize the validation process, mixing the train and validation bar make it quite hard to follow the train process!

Maybe it would be easier to have 2 independent bars (train and val) and a counter for number of validation remaining each epoch?
Or have the possibility to have a third bar for only the training process

**Comment by carmocca** [comment]
I do agree that we could provide an alternative `ProgressBar` that implements the separate behaviour. As many users prefer it that way.

**Comment by kaushikb11** [comment]
Hi @nicolas-dufour, I am closing the issue as it is working as intended. You could create a separate feature request for this if you wish to. That would be appreciated!
